source/zipkin/tracer.cc

```diff
@@ -103,7 +99,7 @@ Span Tracer::startSpan(const std::string& span_name, uint64_t start_time,
   ep.setServiceName(service_name_);
 
   // Add the newly-created annotation to the span
-  annotation.setHost(std::move(ep));
+  annotation.setEndpoint(std::move(ep));
```

Comment:
I suggest using full names instead of shorthands. E.g. endpoint instead of ep. Annotation instead of ann.